sigsep_open-unmix-pytorch_umx.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: Open-Unmix
+summary: Reference implementation for music source separation
+image: sigsep_logo_inria.png
+author: Inria
+tags: [audio, scriptable]
+github-link: https://github.com/sigsep/open-unmix-pytorch
+github-id: sigsep/open-unmix-pytorch
+featured_image_1: sigsep_umx-diagram.png
+featured_image_2: no-image
+accelerator: cuda-optional
+---
+
+
+```bash
+# assuming you have a PyTorch >=1.6.0 installed
+pip install -q torchaudio
+```
+
+```python
+import torch
+
+# loading umxhq four target separator
+separator = torch.hub.load('sigsep/open-unmix-pytorch', 'umxhq')
+
+# audio shape (nb_samples, nb_channels, nb_timesteps)
+# assuming audio is resampled to separator.sample_rate
```

Comment:
how does the user set `separator.sample_rate`? otherwise, can you provide a code snippet to resample, say with torchaudio?

Reply:
that was probably misleading: the user should not set `separator.sample_rate` as this is the rate the model was trained on. 

> otherwise, can you provide a code snippet to resample, say with torchaudio?

sure, what about this?

```python
resampler = torchaudio.transforms.Resample(orig_freq=rate, new_freq=separator.sample_rate, resampling_method="sinc_interpolation").to(audio.device)
audio = resampler(audio)
```

Reply:
How about this?

```suggestion
# first resample to the sample rate of the models, separator.sample_rate
resampler = torchaudio.transforms.Resample(original_sample_rate, separator.sample_rate)
audio = resampler(audio)
```

Since Resampler only has ints/floats/strings attached to it, there shouldn't be a need for moving it explicitly to another device.

btw pytorch/audio#1402 makes the following one-line work on master: 
```python
audio = torchaudio.transforms.resample(audio, original_sample_rate, separator.sample_rate)
```
Writing the transform version as a one-line may be less readable. Thoughts?
```python
audio = torchaudio.transforms.Resample(original_sample_rate, separator.sample_rate)(audio)
```

Reply:
yes, I also prefer the two-liner...